cmd/bnsd/x/blueaccount/msg.go

```diff
@@ -0,0 +1,180 @@
+package blueaccount
+
+import (
+	weave "github.com/iov-one/weave"
+	"github.com/iov-one/weave/errors"
+	"github.com/iov-one/weave/migration"
+)
+
+func init() {
+	migration.MustRegister(1, &UpdateConfigurationMsg{}, migration.NoModification)
+
+	migration.MustRegister(1, &RegisterDomainMsg{}, migration.NoModification)
+	migration.MustRegister(1, &TransferDomainMsg{}, migration.NoModification)
+	migration.MustRegister(1, &RenewDomainMsg{}, migration.NoModification)
+	migration.MustRegister(1, &DeleteDomainMsg{}, migration.NoModification)
+
+	migration.MustRegister(1, &RegisterAccountMsg{}, migration.NoModification)
+	migration.MustRegister(1, &TransferAccountMsg{}, migration.NoModification)
+	migration.MustRegister(1, &ReplaceAccountTargetsMsg{}, migration.NoModification)
+	migration.MustRegister(1, &DeleteAccountMsg{}, migration.NoModification)
+}
+
+var _ weave.Msg = (*UpdateConfigurationMsg)(nil)
+
+func (UpdateConfigurationMsg) Path() string {
+	return "blueaccount/updateconfigurationmsg"
```

Comment:
If you check other extensions message path format you will see usage of underscores such as: https://github.com/iov-one/weave/blob/master/x/cash/msg.go#L131. And also the `msg` suffix is not appended to other extensions.

Reply:
Good catch, I even [documented it](https://github.com/iov-one/weave/blob/ea73c9ce47eed49f6eb03b353202dc34a010e481/tx.go#L23-L27).